elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -597,7 +605,14 @@ export class PeerPool extends EventEmitter {
 	}
 
 	private _selectPeersForEviction(): Peer[] {
-		const peers = [...this.getPeers(InboundPeer)];
+		const peers = [...this.getPeers(InboundPeer)].filter(peer => {
```

Comment:
I'd expand the test coverage for eviction feature.

Reply:
I've tested it in integration test to see if white list from config is making to the sanitized list but since in the sanitized list we remove ourselves from whitelist, we need not local ip to test it. So we can move it to QA testing instead of testing it locally